doc/visual-programming/source/widgets/data/continuize.md

```diff
@@ -17,7 +17,7 @@ The **Continuize** widget receives a data set in the input and outputs the same
 
 1. [Continuization methods](https://en.wikipedia.org/wiki/Continuity_correction), which define the treatment of multivalued discrete attributes. Say that we have a discrete attribute status with the values low, middle and high, listed in that order. Options for their transformation are:
 
-   - **First value as base**: the attribute will be transformed into two continuous attributes, status=middle with values 0 or 1 signifying whether the original attribute had value middle on a particular example, and similarly, status=high. Hence, a three-valued attribute is transformed into two continuous attributes, corresponding to all except the first value of the attribute.
+   - **First value as base**: the attribute will be transformed into one less continuous variable as there are categorical values. The first value alphabetically will be taken as a base. The order of categorical variables can be changed in [Edit Domain](../data/editdomain).
```

Comment:
I think the previous approach to explaining the method using an example (but not necessarily the text itself, e.g. "signifying" should be "indicating") was better.

Instructions for changing the order of variables could be moved to one paragraph earlier.

Reply:
I think docs should first explain in general, what the feature is doing, and second how it applies in a particular use case. Saying 'the attribute will be transformed into two continuous attributes' is false for just about all the attributes where values != 3.

If anything, I think we could keep the example after the general description of the method.

The order of categorical values applies only to this particular method, does it not?

Reply:
> The order of categorical values applies only to this particular method, does it not?

No, also to both methods that treat it as ordinal.